### ISSUE TITLE
Advanced web interface mouse tracking bug with a pinned list column

### ISSUE THREAD

**Issue by Jolly-Friend-9876** [mastodon/mastodon#30557]
### Steps to reproduce the problem

1. Put your Mastodon web interface in advanced web interface.
2. Create a list, and pin it.
3. Once you have more than a few posts, click in the body of the post to show the thread in the right pane.
4. Interact with the thread in the right pane.
5. In the second column of your timeline, try to hover over reply, favorite, or boost.
Observe: You have to move the mouse over screen elements far to the right to highlight the click and to click on the right icon.
6. Click the profile link to show the profile in the right column.
7. Repeat step 5. Observe this time the hover over the icons highlights where your mouse pointer actually is.

### Expected behaviour

Timeline hover should always track to where the mouse pointer is

### Actual behaviour

If you are in thread viewing column, trying to press an icon in the timeline highlights wrong as indicated in the Observe above.

### Detailed description

In a Mastodon release a few months back, I observed on tech.lgbt the above issue. I've been dealing with this since. The frustration is that you cannot rely on where your mouse pointer is in order to click an icon using the above scenario. I just reproduced the same issue on mastodon.social, so I think it is a widespread issue.

### Mastodon instance

mastodon.social

### Mastodon version

v4.3.0-nightly.2024-05-31-security

### Browser name and version

Safari on macOS with latest production version

### Operating system

macOS with latest production version

**Comment by Jolly-Friend-9876** [comment]
tech.lgbt is currently running version "v4.3.0-alpha.4+glitch" by the way, which is where I originally saw this issue.

**Comment by ClearlyClaire** [comment]
Sounds like a duplicate of #29624. I have no idea at the moment what is causing this.

**Comment by Jolly-Friend-9876** [comment]
Yes, I found that other issue and linked it last week. This issue had some reproducible steps the other one did not seem to have.

**Comment by Jolly-Friend-9876** [comment]
Hopefully this significant UI bug can be resolved. I just managed to boost a porn post in my timeline when I was trying to block the user because of what was highlighted relative to where my mouse pointer was. Luckily I was able to unboost, but it was highly embarassing. This control highlighting bug occurs many times per 15 minutes, and I am on Mastodon a lot. Sometimes it isn't off as much to the right, in fact you have to go down and to the left of the UI element sometimes.

Also, sometimes the UI buttons for Like, Reply, etc will not highlight at all, no matter where you move your mouse pointer. The only way to resolve it is to refresh the Mastodon site in the browser. I haven't figured out the pattern for this just yet.
